pixl_imaging/src/pixl_imaging/_orthanc.py

```diff
@@ -113,12 +134,27 @@ async def retrieve_from_remote(self, query_id: str) -> str:
         response = await self._post(
             f"/queries/{query_id}/retrieve",
             data={"TargetAet": self.aet, "Synchronous": False},
+            timeout=self.dicom_timeout,
         )
         return str(response["ID"])
 
-    async def wait_for_job_success_or_raise(
-        self, job_id: str, job_type: str, timeout: float
-    ) -> None:
+    async def retrieve_instances_from_remote(
+        self, modality: str, missing_instances: list[dict[str, str]]
+    ) -> str:
+        """Retieve missing instances from remote modality in a single c-move query."""
+        response = await self._post(
+            f"/modalities/{modality}/move",
+            data={
+                "Level": "Instance",
+                "TargetAet": self.aet,
+                "Synchronous": False,
+                "Resources": missing_instances,
+            },
+            timeout=self.dicom_timeout,
```

Comment:
this one should be the http timeout as its just waiting for the job id from the POST